<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Compare</class>
 <widget class="QWidget" name="Compare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conflict detected in keyfiles.</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="currentLabel">
       <property name="toolTip">
        <string>Current Keyfile Version</string>
       </property>
       <property name="text">
        <string>Current Keyfile Version</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="importedLabel">
       <property name="toolTip">
        <string>Imported Keyfile Version</string>
       </property>
       <property name="text">
        <string>Imported Keyfile Version</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeView" name="currentTreeView">
       <property name="toolTip">
        <string>Keyfile content from the database</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="importedTreeView">
       <property name="toolTip">
        <string>Keyfile content from the disk</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="toolTip">
      <string>Details of the selected file</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="toolTip">
        <string>Close</string>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
